src/Controllers/DisplayEditApplicantController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Portal\Controllers;
+
+use Portal\Abstracts\Controller;
+use Portal\Interfaces\ApplicantModelInterface;
+use Portal\Models\ApplicantModel;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Views\PhpRenderer;
+
+class DisplayEditApplicantController extends Controller
+{
+    private $applicantModel;
+    private $renderer;
+    /**
+     * DisplayEditApplicantController constructor.
+     * @param ApplicantModelInterface $applicantModel
+     * @param PhpRenderer $renderer
+     */
+    public function __construct(ApplicantModelInterface $applicantModel, PhpRenderer $renderer)
+    {
+        $this->applicantModel = $applicantModel;
+        $this->renderer = $renderer;
+    }
+
+    public function __invoke(Request $request, Response $response, array $args)
+    {
+        if ($_SESSION['loggedIn'] === true) {
+            $id = $request->getQueryParams()['id'];
+            $applicant = $this->applicantModel->getApplicantById($id);
+            $data = [
```

Comment:
WHY?! just pass the entity into the view, this is such a waste of time and processing

Reply:
also by doing this, you entirely defeat the point of using an entity, the data could not be edited by someone accidentally

Reply:
https://media.giphy.com/media/U4VXRfcY3zxTi/giphy.gif